src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -127,69 +105,90 @@
       :status          :default
       :size            :small
       :title           (i18n/label :t/max-fees)
-      :subtitle        "€188,70"}]
+      :subtitle        (i18n/label :t/amount-with-currency-symbol
+                                   {:amount (str max-fees)
+                                    :symbol "$"})}]
     [quo/data-item
-     {:container-style {:flex   1
-                        :height 36}
+     {:container-style style/detail-item
       :blur?           false
       :description     :default
       :icon-right?     false
       :card?           false
       :label           :none
       :status          :default
       :size            :small
-      :title           (i18n/label :t/user-gets {:name "Mark"})
-      :subtitle        "149.99 ETH"}]]])
+      :title           (i18n/label :t/user-gets {:name (utils/get-shortened-address to-address)})
+      :subtitle        (str amount " " (:symbol token))}]]])
 
-(defn- f-view-internal
+(defn- view-internal
   [_]
-  (let [reset-slider?        (reagent/atom false)
-        margin-top           (safe-area/get-top)
-        biometric-auth?      true
-        on-close             #(rf/dispatch [:navigate-back-within-stack :wallet-select-asset])
-        status-account-props {:customization-color :purple
-                              :size                32
-                              :emoji               "🍑"
-                              :type                :default
-                              :name                "Collectibles vault"
-                              :address             "0x0ah...78b"}
-        user-props           {:full-name           "M L"
-                              :status-indicator?   false
-                              :size                :small
-                              :ring-background     (resources/get-mock-image :ring)
-                              :customization-color :blue
-                              :name                "Mark Libot"
-                              :address             "0x0ah...78b"
-                              :status-account      (merge status-account-props
-                                                          {:size  16
-                                                           :name  "New house"
-                                                           :emoji "🍔"})}]
+  (let [on-close              #(rf/dispatch [:navigate-back-within-stack :wallet-select-asset])
+        send-transaction-data (rf/sub [:wallet/wallet-send])
+        token                 (:token send-transaction-data)
+        token-symbol          (:symbol token)
+        amount                (:amount send-transaction-data)
+        route                 (:route send-transaction-data)
+        estimated-time-min    (:EstimatedTime route)
```

Comment:
Can the keys in this data be converted to kebab-case

Reply:
yeah you're right, will adjust these!

Reply:
done